server.py

```diff
@@ -92,11 +92,15 @@ def __init__(self, map_name):
             # and assign it to client
             robot = self.assign_robot_to_client(ws)
             # Prepare message to send: robot name, game state and cards
-            welcome_message = {"robot_name": robot.name,
-                               **self.state.whole_as_dict(map_name),
-                               **self.state.cards_and_game_round_as_dict(
-                               robot.dealt_cards, robot.select_blocked_cards_from_program()),
-                               }
+            welcome_message = {
```

Comment:
@befeleme ty blokované karty jsou na začátku jen prázdný seznam, chceš měnit tu metodu? Já to dala právě do jedné zprávy, chtěla jsem ať se posílá vše naráz.

Reply:
jde mi o to, ze welcome message je zprava, ktera se posila pouze jednou - po pripojeni klienta. v tu chvili jeste nema zadne blokovane karty, nemusime mu je tak posilat. v dalsich kolech posilame karty a informace z jineho mista.

Reply:
upravim to, az budu mit chvilku a poslu do tohoto PR.

Reply:
Az ted vidim, o co jde! Spatne jsem to celou dobu interpretovala. To potom vyresime upravou metody, jak jsme vcera komentovaly. Tak jo, upravim komentar a bude v tomto PR hotovo.